packages/main/src/plugin/kubernetes/kubernetes-port-forward-service.ts

```diff
@@ -86,7 +91,9 @@ export class KubernetesPortForwardService implements IDisposable {
    * @see UserForwardConfig
    */
   async createForward(config: UserForwardConfig): Promise<ForwardConfig> {
-    return this.configManagementService.createForward(config);
+    const forwardConfig = await this.configManagementService.createForward(config);
+    this.apiSender.send('kubernetes-port-forwards-update');
```

Comment:
To have similar update pattern than the kubernetes resources, we should provide the array in the event 
```suggestion
    this.apiSender.send('kubernetes-port-forwards-update', this.listForwards());
```